### DISCUSSION TITLE
Admin email is not a valid email - user@localhost

### DISCUSSION THREAD

**Discussion by milahu** [requarks/wiki#5949] [Error / Bug Report] [answered]
me, trying to install wiki.js on localhost ...

wiki: Administrator Email?
me: user@localhost
wiki: Admin email is not a valid email
me: user@localhost.lan
wiki: ok

**Comment by NGPixel** [comment, marked as answer]
Even though "localhost" is technically a valid hostname, it's a very edge case scenario. The current validation is there to catch invalid email for most users running a public production instance.

Use a publicly valid email address.
